### PR TITLE
docs: update docker commands to start containers in detached mode

### PR DESCRIPTION
**Pull request by Clivern**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,19 +129,24 @@ The are numbers of ways you can run Unleash.
 
 ### Run from Docker
 
-1. Create a network by running docker network create unleash
+1. Create a network by running the following command:
+
+```sh
+docker network create unleash
+```
+
```
Comment (thomasheartman) [inline README.md:137]: This is a great catch; thanks! 🎉

File: README.md [hunk 2]
```diff
@@ -129,19 +129,24 @@ The are numbers of ways you can run Unleash.
...
 2. Start a postgres database:
 
 ```sh
-docker run -e POSTGRES_PASSWORD=some_password \
+docker run -d -e POSTGRES_PASSWORD=some_password \
```
Comment (thomasheartman) [inline README.md:141]: For the readme, we'd like to keep the command examples blocking. This makes it easier for people who aren't used to Docker to get started. See my general review comment for a more thorough discussion ☺️ 

```suggestion
docker run -e POSTGRES_PASSWORD=some_password \
```

File: README.md [hunk 3]
```diff
@@ -129,19 +129,24 @@ The are numbers of ways you can run Unleash.
...
   -e POSTGRES_USER=unleash_user -e POSTGRES_DB=unleash \
   --network unleash --name postgres postgres
 ```
 
 3. Start Unleash via docker:
 
 ```sh
-docker run -p 4242:4242 \
+docker run -d -p 4242:4242 \
```
Comment (thomasheartman) [inline README.md:149]: As above, we'd like to keep readme sample commands blocking. See the general comment for more on that 😄 

```suggestion
docker run -p 4242:4242 \
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Thanks for the PR 🙏🏼  

Good job spotting the unfenced network creation command and giving it a proper code block!

Regarding running docker commands in detached mode, we'd like to keep the readme commands blocking for a few reasons:

1. **It's the minimum command**. There's nothing in the command at the moment that is optional. The `-d` option isn't needed and provides more 'noise'.

2. **It makes it easier for people new to Docker**. If you're not familiar with docker, it can be a bit confusing when it just outputs the container hash and returns you to your prompt. What just happened? How do I see what's happening in the container?

3. **Makes it easier to inspect**. You _can_ connect to a detached container, but it requires extra steps. As the readme is intended to facilitate learning, it makes sense that you should be able to inspect the container's output by default.

And finally, if you're comfortable enough with docker to want to run it detached, then you'll be able to add the required options easily enough yourself.

However, I'd be very happy to hear your reasons for using the `-d` option in the readme, and we can certainly discuss it further. For now, I've added the old commands as suggestions.

Let me know what you think ☺️

**Comment by Clivern** [comment]
@thomasheartman I use detached based on personal preference so i would agree with you. PR got updated

**Review by thomasheartman — APPROVED** [review]

**Comment by thomasheartman** [comment]
Thank you! 😄  I'll merge it in now 🙋🏼

**Pull request merged**